src/deluge/gui/views/view.cpp

```diff
@@ -940,10 +940,6 @@ void View::displayModEncoderValuePopup(Param::Kind kind, int32_t paramID, int32_
 			popupMsg.append(name);
 			popupMsg.append(": ");
 		}
-		else {
```

Comment:
Instead of removing the append for midi, can we replace it with an append for Midi CC #?

For Midi instruments the cc # = modelStackWithParam->paramId

So:

popupMsg.append("CC ";
popupMsg.appendInt(modelStackWithParam->paramId);
popupMsg.append(": ");

Reply:
Yes, but you'd have to check the Param kind == MIDI, but I'd really rather do it in getDisplayName ForParam